cmd/shellwrapper.go

```diff
@@ -67,6 +67,26 @@ konf_cleanup() {
   konf-go cleanup
 }
 
+trap konf_cleanup EXIT
+`
+
+	var fish = `
+function konf -w konf-go
+    set -f res (konf-go $argv)
+    # only change $KUBECONFIG if instructed by konf-go
+    if string match -q 'KUBECONFIGCHANGE:*' $res
+        # this basically takes the line and cuts out the KUBECONFIGCHANGE Part
+        set -gx KUBECONFIG (string replace -r '^KUBECONFIGCHANGE:' '' $res)
+    else
+        # this makes --help work
+        printf "%s\n" $res
```

Comment:
nit: I think it would be a good idea to add a tiny comment above here that says: "Because fish does not support bracketed vars, we use printf instead"

Reply:
I've added the line